src/init.cpp

```diff
@@ -1567,7 +1559,7 @@ bool AppInitMain(boost::thread_group& threadGroup, CScheduler& scheduler)
 
     // ********************************************************* Step 8: load wallet
 #ifdef ENABLE_WALLET
-    if (!InitLoadWallet())
+    if (!OpenWallets())
```

Comment:
Can you add the rationale for rename in the commit message? Why not keep it?

Reply:
@jnewbery Or add it to the OP, so it can be seen in the merge commit.

Reply:
done!